docs/sigs/Cloud.md

```diff
@@ -20,7 +20,7 @@ We do not currently hold regular meetings, but work asynchronously in mattermost
 ## Activities, projects, and deliverables
 
 * The AlmaLinux OS Docker/Podman images:
```

Comment:
Can we replace _Docker/Podman_ with just "Container" word?

Reply:
If it's equally accurate, I think that's a good idea! @sboldyreva what do you think?